domain/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+plugins {
+    id("java-library")
+    id("org.jetbrains.kotlin.jvm")
+}
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_7
```

Comment:
@BottleRocket-Colin A couple of things:

1. In order to use java 8 Date/Time apis, it seems this will need to be increased to at least java 8. For the app and data modules, we're using java 11 (like below):

```
        sourceCompatibility = JavaVersion.VERSION_11
        targetCompatibility = JavaVersion.VERSION_11
```

Any reason to not use java 11 here for consistency?

2. We also specify the jvm target to kotlin in the app and data modules (as seen below):

```
kotlinOptions {
        jvmTarget = JavaVersion.VERSION_11.toString()
    }
```

a. Should we also be doing this for the java library?
b. If so, is `kotlinOptions` the proper syntax to use and is it available at top level? Note that it is nested under the AGP plugin `android` block for the app and data modules.

Reply:
Will update in next PR.